<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存在(I):</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon01">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon02">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon03">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon04">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="desktop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>桌面</string>
   </property>
  </widget>
  <widget class="QLabel" name="docs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我的文档</string>
   </property>
  </widget>
  <widget class="QLabel" name="computer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我的电脑</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名(M):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fileNameComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存类型(T):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fileTypeComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="saveComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
